.github/workflows/R-CMD-check.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
 
       # Step 3: Cache R packages
       - name: Cache R packages
-        uses: actions/cache@v2
+        uses: actions/cache@v3
```

Comment:
This step of the github action had `v2` become depreciated this week so it made the GHA fail.  Switching to `v3` fixed it.